searchkit/search.py

```diff
@@ -977,17 +978,27 @@ def _run_search(self, fd):
         self.stats.reset()
         sequence_results = SequenceSearchResults()
         search_ids = set([s.id for s in self.search_defs])  # noqa, pylint: disable=R1718
+        search_tags = set([s.tag for s in self.search_defs]) # noqa, pylint: disable=R1718
         offset = self.constraints_manager.apply_global(search_ids, fd)
-        log.debug("starting search of %s (offset=%s, pos=%s)", fd.name, offset,
-                  fd.tell())
+        log.debug("starting search of %s (offset=%s, pos=%s) for tags [%s]",
+                  fd.name, offset, fd.tell(), search_tags)
```

Comment:
it is nice to see the tags but maybe we need to format it differently to make it readable since it will be a very very long line otherwise.

Reply:
done:

```
2024-05-09 15:20:57,660 544681 DEBUG searchkit [-] starting search of /workspace/case-data/test/sosreport-juju-4b6be0-3-lxd-13-00359298-2023-05-02-tiaplar/var/log/neutron/neutron-server.log.1 (offset=0, pos=0) for tags [[
  "openstack.neutron.ovn_stale_db.checks.has_1829109_neutron_log.check.search",
  "openstack.neutron.bugs.lp1948466.checks.has_1948466.check.search",
  "openstack.neutron.bugs.lp1980211.checks.has_ovsdb_reconnection_errors.check.search",
  "openstack.neutron.bugs.lp1960319.checks.has_1960319_log.check.search",
  "openstack.neutron.bugs.lp1993628.checks.logs.check.search",
  "openstack.neutron.ovndb_leader_bouncing.checks.has_nbdb_log.check.search",
  "openstack.neutron.ovndb_leader_bouncing.checks.has_sbdb_log.check.search"
]]
```